src/app/lib/config/services/ws.ts

```diff
@@ -258,6 +258,16 @@ export const service: DefaultServiceConfig = {
           lang: 'en-GB',
         },
       ],
+      extraLinks: [
+        {
+          href: 'https://bbcnews.bbcstudios.com',
+          text: 'BBC Studios Commercial Opportunities',
+        },
+        {
+          href: 'https://www.bbc.co.uk/programmes/articles/2x9tqt6mc05vB2S37j8MWMJ/global-short-wave-frequencies',
```

Comment:
Let's use .com instead
```suggestion
          href: 'https://www.bbc.com/programmes/articles/2x9tqt6mc05vB2S37j8MWMJ/global-short-wave-frequencies',
```